server/app/attestation-service-ctl/src/api.rs

```diff
@@ -0,0 +1,159 @@
+use anyhow::Result;
+use clap::{App, Arg, SubCommand};
+use shadow_rs::shadow;
+use std::path::Path;
+
+mod attestation;
+mod management;
+
+pub mod management_api {
+    tonic::include_proto!("management");
+}
+pub mod attestation_api {
+    tonic::include_proto!("attestation");
+}
+pub mod common {
+    tonic::include_proto!("common");
+}
+
+#[macro_use]
+extern crate log;
+shadow!(build);
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    env_logger::builder()
+        .filter(None, log::LevelFilter::Info)
+        .init();
+
+    let version = format!(
+        "\nv{}\ncommit: {}\nbuildtime: {}",
+        build::PKG_VERSION,
+        build::COMMIT_HASH,
+        build::BUILD_TIME
+    );
+
+    let matches = App::new("client")
```

Comment:
FWIW, `clap`allows for a very convenient, `derive`-based definitions of commands, subcommands and their related arguments. Here is a good example: https://github.com/clap-rs/clap/blob/v3.2.6/examples/git-derive.rs

Most of the parsing you're doing below would no longer be necessary with that approach.